install.sh

```diff
@@ -1,6 +1,16 @@
 #!/usr/bin/env bash
 set -e
 
-python setup.py install --force --user --prefix=
+which profanity > /dev/null
+which scanelf > /dev/null
+which tail > /dev/null
+which cut > /dev/null
+which tr > /dev/null
+which sed > /dev/null
+
+profanity="$(which profanity)"
+python_version="$(scanelf -n "${profanity}" | tail -n+2 | cut -d' ' -f2 | tr , '\n' | sed -nre 's/^libpython([0-9]+\.[0-9]+).*$/\1/p')"
```

Comment:
`ldd` can be used to execute arbitrary code: http://www.catonmat.net/blog/ldd-arbitrary-code-execution/

Hence I don't use `ldd` (which more or less executes the binary), but `scanelf` (which simply parses the ELF header). Another alternative would something like the following: `readelf -d ${profanity} | awk '$2~/NEEDED/{gsub(/[\[\]]/, "", $5) ; print($5)}'`, which is more complex, though.

Using Python would be preferable, but you'd need to load a package that parses the ELF header, which would be an additional build-time dependency.